tests/lib/service-state.test.js

```
@@ -0,0 +1,96 @@
+import { describe, expect, test } from 'bun:test';
+
+const {
+  evaluateServiceState,
+  waitForServiceReadiness,
+} = require('../../src/lib/service-state.cjs');
```

ESLint

[error] 'require' is not defined.

(no-undef)